### PR TITLE
Fix edit and destroy bug, will test later

### PR DESCRIPTION
**Pull request by ericywong**
tada!

### CODE REVIEW

**Comment by MSevey** [comment]
i think this is ok at best

File: app/views/tweets/index.html.erb [hunk 1]
```diff
@@ -17,8 +17,10 @@
         <td><%= tweet.content %></td>
         <td><%= link_to tweet.user.email, show_user_path(tweet.user) %></td>
         <td><%= link_to 'Show', tweet %></td>
-        <td><%= link_to 'Edit', edit_tweet_path(tweet) %></td>
-        <td><%= link_to 'Destroy', tweet, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+        <% if user_signed_in? && (tweet.user_id == current_user.id) %>
```
Comment (nw0428) [inline app/views/tweets/index.html.erb]: This seems like a method that belongs in tweet! Maybe `Tweet#owned_by?(owner)`?

**Review by nw0428 — CHANGES_REQUESTED** [review]
DRY this out please

**Review by MSevey — DISMISSED** [review]
good, there are more opportunities for links to not show.  go find them

**Review by MSevey — CHANGES_REQUESTED** [review]
There is another link that should not be seen all the time.  Can't follow yourself

**Review by nw0428 — APPROVED** [review]
AWESOME. Merge away

**Review by MSevey — APPROVED** [review]

**Pull request merged**